tools/analyzer_plugin/test/integration/diagnostics/analysis_options_configuration_test.dart

```diff
@@ -63,6 +63,15 @@ void main() {
       final errors = await computeErrors();
       expectSeverity(errors, AnalysisErrorSeverity.ERROR);
     });
+
+    // This test ensures that when one of the error codes for a diagnostic contributor is disabled, it doesn't prevent
+    // the other error codes from being computed.
+    test('configured with partially disabled diagnostic contributor', () async {
+      await setUp(yamlContents: yamlPartiallyDisabledContributor);
+      final errors = await computeErrors(sourceCode: badKeysSourceCode);
+      expect(errors, hasLength(1));
```

Comment:
Can we assert somewhere that this code yields two keys when the contributor is not partially or fully disabled, to ensure this test's assumptions don't become incorrect silently if the implementation of these lints changes?

Reply:
Done